packages/ui-color-picker/src/ColorMixer/ColorPalette/index.tsx

```diff
@@ -234,8 +238,11 @@ class ColorPalette extends Component<ColorPaletteProps, ColorPaletteState> {
         tabIndex={this.props.disabled ? undefined : 0}
         onKeyDown={(e) => this.handleKeyDown(e)}
         onMouseDown={(e) => this.handlePaletteMouseDown(e)}
-        aria-label={this.props.navigationExplanationScreenReaderLabel}
+        id={this._id}
       >
+        <ScreenReaderContent>
```

Comment:
As we discussed on the standup: please ask Jes about this :)

Reply:
I've just asked her